TOC.md

```diff
@@ -140,9 +140,8 @@
     - [TiSpark Topology](/tispark-deployment-topology.md)
     - [Cross-DC Topology](/geo-distributed-deployment-topology.md)
     - [Hybrid Topology](/hybrid-deployment-topology.md)
-  - Install and Start
-    - [Use TiUP](/production-deployment-using-tiup.md)
-    - [Deploy on Kubernetes](/tidb-in-kubernetes.md)
+  - [Deploy Using TiUP](/production-deployment-using-tiup.md)
```

Comment:
Deploy **with** TiUP? Also the URL.

Reply:
Generally, we limit the use of the word "with" in documentation because it's not clear enough. Ref: https://developers.google.com/style/word-list#with